target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -162,6 +162,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "1:lan" "2:lan" "3:wan" "6@eth0"
 		;;
+	tuoshi,lt22m)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "6t@eth0"
```

Comment:
I tried ":0" and ":1" before (because that's the label on the pcb, I saw later that the case has 1 and 2). But it had no effect to the labels in luci's "switch" page.

Reply:
Hmm, again it's a `swconfig` thing which I'm not familiar with.

Reply:
Maybe `"1:lan" "0:lan" "6t@eth0"`?

Reply:
makes no sense to swap both. I'll add the :1 and :2